### PR TITLE
Fixed DFDL byte pos event seeking in viewport

### LINKED ISSUE #967: Byte Position Message Does Not Properly Seek Outside Viewport
I ran into this problem while running through the additional 1.4.0 tests for the data editor. The byte position message does not properly traverse to the appropriate viewport segment when the position is outside of the current viewport.

### PR DESCRIPTION
**Pull request by stricklandrbls**
- Created a Svelte store to maintain the bytePos1b data received from the DFDL debug runtime then calculate it's highlighting position from the viewport file offset.

Closes #967

### CODE REVIEW

**Review by rthomas320 — APPROVED** [review]
I opened the gif dfdl schema file in the debugger, without opening the data editor.  I stepped forward in the schema file to a point where the schema was reading color block data from the image. Then, I opened the debug editor to the same data file.  I stepped forward in the schema file and the indicator in the data editor highlighted the appropriate bytes in the data file as I step forward.

File: src/svelte/src/components/Main.svelte [hunk 1]
```diff
@@ -32,7 +32,7 @@ limitations under the License.
       on:scrollBoundary
       on:traverse-file
       on:seek
-    />    
+          />
```
Comment (scholarsmate) [inline src/svelte/src/components/Main.svelte:35]: Weird whitespace change.

File: src/svelte/src/components/dataEditor.svelte [hunk 2]
```diff
@@ -153,7 +152,7 @@ limitations under the License.
   function seekEventHandler(_: CustomEvent) {
     seek($seekOffset)
   }
-
+
```
Comment (scholarsmate) [inline src/svelte/src/components/dataEditor.svelte:155]: Weird whitespace change.

File: src/svelte/src/components/DataDisplays/DataViewports.svelte [hunk 3]
```diff
@@ -27,7 +27,7 @@ limitations under the License.
 <DataLineFeed
   on:traverse-file
   on:seek
-  on:applyChanges
+    on:applyChanges
```
Comment (scholarsmate) [inline src/svelte/src/components/DataDisplays/DataViewports.svelte:30]: Weird whitespace change.

**Review by scholarsmate — APPROVED** [review]
+1, looks good aside from some weird whitespace changes.

**Comment by stricklandrbls** [comment]
@scholarsmate I must have whitespace git changes turned off somewhere. Those whitespace changes don't appear in my changes, even when I soft reset to main.

**Pull request merged**